lametro/bills.py

```diff
@@ -171,7 +171,7 @@ def scrape(self, window=28, matter_ids=None) :
                 # required fields
                 bill.title = 'Restricted View'
                 bill.add_subject('Restricted View')
-                bill.add_source('https://metro.legistar.com')
+                bill.add_source('https://metro.legistar.com', note='web')
```

Comment:
why can't we add the API source?

Reply:
does it matter? 

I am adding the web source here, so that I can minimize the number of changes needed to fix the Councilmatic errors, i.e., [I'll need to adjust the `import_data` script, since that only adds a source_url if the note says "web"](https://github.com/datamade/django-councilmatic/blob/master/councilmatic_core/management/commands/import_data.py#L969)

Reply:
it may matter for debugging purposes so we can find the bill in the source system. 

if we are hacking around the import_data script, i think it's probably better to put in the api url and lie about it being a web url. (with a comment here to that effect)

Reply:
I see. 

I am more inclined to _not_ lie about the url and, in addition to this PR, fix the import_data script.

Reply:
that seems reasonable.